pkg/exchange/okex/stream.go

```diff
@@ -343,8 +342,11 @@ func (s *Stream) read(ctx context.Context) {
 					s.EmitEvent(*et)
 
 				case *BookData:
-					s.EmitBookData(*et)
-
+					//there's "books" for 400 depth and books5 for 5 depth
+					if channel != "books5" {
+						s.EmitBookData(*et)
+					}
+					s.EmitBookTickerUpdate(et.BookTicker())
```

Comment:
should EmitBookTickerUpdate only for `books5`?

Reply:
it should all fulfill the rule of bookticker in my though,but i could change it .